<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>181</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255) </string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QRadioButton" name="AvtoRadioButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Автоматический</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="ManualRadioButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ручной</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Выберите режим</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="AvtoBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>161</width>
       <height>101</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="statusTip">
      <string>Окошечко автоматического режима работы программы</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="title">
      <string>Автоматический</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Кол-во шагов :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NumStepsAvto">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="AvtoBoxButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Применить</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>801</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 255, 255);
QPushButton
{
	height: 30;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="ReturnBackButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(85, 255, 127)</string>
     </property>
     <property name="text">
      <string>Вернуться на шаг назад</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveImageButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>20</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(85, 255, 127)</string>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OneStepButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>191</width>
       <height>38</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background: rgb(147, 167, 255)
}
QPushButton:hover
{
	background:rgb(255, 0, 0)
}</string>
     </property>
     <property name="text">
      <string>1 шаг</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TenStepButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>190</width>
       <height>38</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background: rgb(147, 167, 255)
}
QPushButton:hover
{
	background:rgb(255, 0, 0)
}</string>
     </property>
     <property name="text">
      <string>10 шагов</string>
     </property>
    </widget>
    <widget class="QPushButton" name="HundredStepButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>191</width>
       <height>38</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background: rgb(147, 167, 255)
}
QPushButton:hover
{
	background:rgb(255, 0, 0)
}</string>
     </property>
     <property name="text">
      <string>100 шагов</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ThousandStepButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>190</width>
       <height>38</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(147, 167, 255)</string>
     </property>
     <property name="text">
      <string>1000 шагов</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Image">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="TakeImage"/>
    <addaction name="SaveImage"/>
    <addaction name="ExitMenu"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="Help"/>
    <addaction name="Education"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="TakeImage">
   <property name="text">
    <string>Выбрать изображение</string>
   </property>
  </action>
  <action name="SaveImage">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="ExitMenu">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="Help">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="Education">
   <property name="text">
    <string>Обучение</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
